### ISSUE TITLE
webcam doesn't work for Chrome on insecure origins.

### ISSUE THREAD

**Issue by cjwelborn** [ejci/favico.js#108]
Trying the webcam demo on http://lab.ejci.net/favico.js/ doesn't do anything for Chrome 48.0.2564.82.
The button text changes (both buttons), but the icon doesn't change and I'm never asked for permission to use the webcam. `Stop webcam` seems to refresh the page. 

Console output:
```
favicon.min.js:7

getUserMedia() no longer works on insecure origins.
To use this feature, you should consider switching your application to a secure origin, such as HTTPS.
See https://goo.gl/rStTGz for more details.
```

**Comment by asiminiceanu** [comment]
That is normal behavior since last December on Chrome for getUserMedia.

**Comment by ejci** [comment]
Yep. Its a chrome behavior (which was a good decision from them to make it like that).
I will try to update the page to work on https.

**Comment by cjwelborn** [comment]
Sorry, for not being clear. It was late when I filed this. Updating the demo page to HTTPS was my suggestion (from the console output). I just wanted to include the behavior that I saw. I switched to HTTPS a while back using StartCom. My host is WebFaction, and they made it pretty painless. The big browsers are making it harder and harder to get away with HTTP (with good reason I think). I don't believe that *every* site *has* to be encrypted, but it would probably be better if they were.
